From a96ac44091b6d8b9fc4b65477117d9a568687b23 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Mon, 23 Oct 2023 10:38:12 -0700
Subject: [PATCH 03/12] Do not reuse mock group

As the mock group might be defined in the nested chroot, do not reuse
it.

Rebased for mock 2.4 update by Francisco Boni <boboniboni@gmail.com>.
Rebased for mock 2.12 update by Patrick McCarty.
Rebased for mock 3.0 update by Patrick McCarty.
Rebased for mock 4.0 update by William Douglas.
Rebased for mock 5.2 update by William Douglas.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
Signed-off-by: William Douglas <william.douglas@intel.com>
---
 py/mockbuild/config.py | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/py/mockbuild/config.py b/py/mockbuild/config.py
index 7e72c68..1aeed5d 100644
--- a/py/mockbuild/config.py
+++ b/py/mockbuild/config.py
@@ -64,11 +64,7 @@ def setup_default_config_opts():
     config_opts['rpmbuild_timeout'] = 0
     config_opts['host_arch'] = os.uname()[-1]
     config_opts['chrootuid'] = os.getuid()
-    try:
-        config_opts['chrootgid'] = grp.getgrnam("mock")[2]
-    except KeyError:
-        #  'mock' group doesn't exist, must set in config file
-        pass
+    config_opts['chrootgid'] = os.getgid()
     config_opts['chrootgroup'] = 'mock'
     config_opts['chrootuser'] = 'mockbuild'
     config_opts['build_log_fmt_name'] = "unadorned"
-- 
2.42.0

